src/clr-angular/utils/i18n/common-strings.service.ts

```diff
@@ -46,6 +46,9 @@ export class ClrCommonStringsService implements ClrCommonStrings {
   signpostToggle = 'Signpost Toggle';
   signpostClose = 'Signpost Close';
   loading = 'Loading';
+  singleSelectionAriaLabel = 'Single selection header';
+  singleActionableAriaLabel = 'Single actionable header';
+  detailExpandableAriaLabel = 'Expandable Row';
```

Comment:
This is probably not the best, it would be better to day 'Toggle more row content', since that is what the button does.